service/src/test/java/bio/terra/workspace/service/resource/controlled/cloud/azure/AzureStorageAccessServiceUnitTest.java

```diff
@@ -490,47 +461,34 @@ public void createAzureStorageContainerSasToken_LzSharedStorageAccountNotFound_F
 
   /** Set up mocks behavior based on testing scenario */
   private void setupMocks(
-      ControlledAzureStorageResource storageAccountResource,
       ControlledAzureStorageContainerResource storageContainerResource,
-      Optional<Boolean> lzSharedStorageAccountExists) {
+      boolean lzSharedStorageAccountExists) {
```

Comment:
Isn't `lzSharedStorageAccountExists` always true now, so this variable can be removed entirely?